.npmignore

```diff
@@ -0,0 +1,10 @@
+.sass-cache/
```

Comment:
No strong opinion. Actually, at first I had this building into `examples/`, but since that's not the case, I don't think I explicitly need to add this.